singhnitin372/api-automation
# Fixed the verify data check for post, put and delete posts test cases

Test case changed
1. verify data after calling create new posts service
2. verify data after calling update new posts service
3. verify data after calling delete new posts service
